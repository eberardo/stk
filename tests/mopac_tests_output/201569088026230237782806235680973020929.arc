

                     SUMMARY OF PM7 CALCULATION, Site No: 23169

                                                       MOPAC2016 (Version: 16.175M)
                                                       Wed May 10 12:35:58 2017
                                                       No. of days remaining =  45

           Empirical Formula: C12 H12 N2  =    26 atoms

PM7 NOOPT  EPS=80.1  CHARGE=0
 201569088026230237782806235680973020929



     NO PARAMETERS MARKED FOR OPTIMIZATION, SO 1SCF WAS USED  
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =         31.88946 KCAL/MOL =     133.42551 KJ/MOL
          TOTAL ENERGY            =      -2008.75443 EV
          ELECTRONIC ENERGY       =      -2929.93505 EV
          CORE-CORE REPULSION     =        921.18062 EV
          DIELECTRIC ENERGY       =         -0.66114 EV
          DIPOLE                  =          1.56851 DEBYE    POINT GROUP:       C2  
          NO. OF FILLED LEVELS    =         35
          IONIZATION POTENTIAL    =          8.516218 EV
          HOMO LUMO ENERGIES (EV) =         -8.516 -0.238
          MOLECULAR WEIGHT        =        184.2402
          COSMO AREA              =        227.07 SQUARE ANGSTROMS
          COSMO VOLUME            =        234.69 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H    22    H    15    10.93170
            H    20    H    26     4.74449
            H    23    H    19     2.36150
          SCF CALCULATIONS        =          2
          WALL-CLOCK TIME         =          0.020 SECONDS
          COMPUTATION TIME        =          0.060 SECONDS


          FINAL GEOMETRY OBTAINED
PM7 NOOPT  EPS=80.1  CHARGE=0
 201569088026230237782806235680973020929

  N     1.23900000 +0   1.62230000 +0   4.91650000 +0
  C     0.91800000 +0   1.38440000 +0   3.57550000 +0
  C     0.11160000 +0   0.30140000 +0   3.22540000 +0
  C    -0.31250000 +0   0.12190000 +0   1.90310000 +0
  C     0.02530000 +0   1.04570000 +0   0.90310000 +0
  C    -0.41890000 +0   0.86120000 +0  -0.48780000 +0
  C    -1.76990000 +0   0.64560000 +0  -0.79740000 +0
  C    -2.18840000 +0   0.46990000 +0  -2.12200000 +0
  C    -1.27640000 +0   0.54630000 +0  -3.17470000 +0
  N    -1.67960000 +0   0.25290000 +0  -4.48200000 +0
  C     0.07660000 +0   0.69900000 +0  -2.87180000 +0
  C     0.49880000 +0   0.87650000 +0  -1.54870000 +0
  C     0.78640000 +0   2.16300000 +0   1.27760000 +0
  C     1.20740000 +0   2.33960000 +0   2.60120000 +0
  H     1.28280000 +0   0.78040000 +0   5.48180000 +0
  H     2.05370000 +0   2.21430000 +0   5.04320000 +0
  H    -0.19390000 +0  -0.42010000 +0   3.97820000 +0
  H    -0.91330000 +0  -0.75200000 +0   1.66010000 +0
  H    -2.51420000 +0   0.61210000 +0  -0.00450000 +0
  H    -3.23900000 +0   0.27650000 +0  -2.32040000 +0
  H    -2.65460000 +0   0.46650000 +0  -4.66670000 +0
  H    -1.06140000 +0   0.62750000 +0  -5.19450000 +0
  H     0.82060000 +0   0.68050000 +0  -3.66350000 +0
  H     1.56110000 +0   1.00230000 +0  -1.35010000 +0
  H     1.03820000 +0   2.92380000 +0   0.54170000 +0
  H     1.76420000 +0   3.23580000 +0   2.86110000 +0
 
