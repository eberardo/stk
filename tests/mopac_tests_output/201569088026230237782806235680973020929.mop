PM7 NOOPT  EPS=80.1  CHARGE=0 
201569088026230237782806235680973020929

N   1.239   +1  1.6223   +1  4.9165   +1 
C   0.918   +1  1.3844   +1  3.5755   +1 
C   0.1116   +1  0.3014   +1  3.2254   +1 
C   -0.3125   +1  0.1219   +1  1.9031   +1 
C   0.0253   +1  1.0457   +1  0.9031   +1 
C   -0.4189   +1  0.8612   +1  -0.4878   +1 
C   -1.7699   +1  0.6456   +1  -0.7974   +1 
C   -2.1884   +1  0.4699   +1  -2.122   +1 
C   -1.2764   +1  0.5463   +1  -3.1747   +1 
N   -1.6796   +1  0.2529   +1  -4.482   +1 
C   0.0766   +1  0.699   +1  -2.8718   +1 
C   0.4988   +1  0.8765   +1  -1.5487   +1 
C   0.7864   +1  2.163   +1  1.2776   +1 
C   1.2074   +1  2.3396   +1  2.6012   +1 
H   1.2828   +1  0.7804   +1  5.4818   +1 
H   2.0537   +1  2.2143   +1  5.0432   +1 
H   -0.1939   +1  -0.4201   +1  3.9782   +1 
H   -0.9133   +1  -0.752   +1  1.6601   +1 
H   -2.5142   +1  0.6121   +1  -0.0045   +1 
H   -3.239   +1  0.2765   +1  -2.3204   +1 
H   -2.6546   +1  0.4665   +1  -4.6667   +1 
H   -1.0614   +1  0.6275   +1  -5.1945   +1 
H   0.8206   +1  0.6805   +1  -3.6635   +1 
H   1.5611   +1  1.0023   +1  -1.3501   +1 
H   1.0382   +1  2.9238   +1  0.5417   +1 
H   1.7642   +1  3.2358   +1  2.8611   +1 
