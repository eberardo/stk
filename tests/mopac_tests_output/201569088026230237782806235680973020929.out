 *******************************************************************************
 ** Site#: 23169        For non-commercial use only     Version 16.175M 64BITS**
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 16.175M, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left:  45**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Wed May 10 12:35:58 2017  *
 *  PM7        - The PM7 Hamiltonian to be used
 *  NOOPT      - DO NOT OPTIMIZE ANY COORDINATES
 *
 *
 *
 *                 CHARGE ON SYSTEM = 0
 *
 *
 *
 *  EPS= 80.10 - USE ANDREAS KLAMT'S COSMO IMPLICIT SOLVATION MODEL
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
PM7 NOOPT  EPS=80.1  CHARGE=0
 201569088026230237782806235680973020929

   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       N          1.23900000      1.62230000      4.91650000   
     2       C          0.91800000      1.38440000      3.57550000   
     3       C          0.11160000      0.30140000      3.22540000   
     4       C         -0.31250000      0.12190000      1.90310000   
     5       C          0.02530000      1.04570000      0.90310000   
     6       C         -0.41890000      0.86120000     -0.48780000   
     7       C         -1.76990000      0.64560000     -0.79740000   
     8       C         -2.18840000      0.46990000     -2.12200000   
     9       C         -1.27640000      0.54630000     -3.17470000   
    10       N         -1.67960000      0.25290000     -4.48200000   
    11       C          0.07660000      0.69900000     -2.87180000   
    12       C          0.49880000      0.87650000     -1.54870000   
    13       C          0.78640000      2.16300000      1.27760000   
    14       C          1.20740000      2.33960000      2.60120000   
    15       H          1.28280000      0.78040000      5.48180000   
    16       H          2.05370000      2.21430000      5.04320000   
    17       H         -0.19390000     -0.42010000      3.97820000   
    18       H         -0.91330000     -0.75200000      1.66010000   
    19       H         -2.51420000      0.61210000     -0.00450000   
    20       H         -3.23900000      0.27650000     -2.32040000   
    21       H         -2.65460000      0.46650000     -4.66670000   
    22       H         -1.06140000      0.62750000     -5.19450000   
    23       H          0.82060000      0.68050000     -3.66350000   
    24       H          1.56110000      1.00230000     -1.35010000   
    25       H          1.03820000      2.92380000      0.54170000   
    26       H          1.76420000      3.23580000      2.86110000   
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         N        1.2390    1.6223    4.9165
     2         C        0.9180    1.3844    3.5755
     3         C        0.1116    0.3014    3.2254
     4         C       -0.3125    0.1219    1.9031
     5         C        0.0253    1.0457    0.9031
     6         C       -0.4189    0.8612   -0.4878
     7         C       -1.7699    0.6456   -0.7974
     8         C       -2.1884    0.4699   -2.1220
     9         C       -1.2764    0.5463   -3.1747
    10         N       -1.6796    0.2529   -4.4820
    11         C        0.0766    0.6990   -2.8718
    12         C        0.4988    0.8765   -1.5487
    13         C        0.7864    2.1630    1.2776
    14         C        1.2074    2.3396    2.6012
    15         H        1.2828    0.7804    5.4818
    16         H        2.0537    2.2143    5.0432
    17         H       -0.1939   -0.4201    3.9782
    18         H       -0.9133   -0.7520    1.6601
    19         H       -2.5142    0.6121   -0.0045
    20         H       -3.2390    0.2765   -2.3204
    21         H       -2.6546    0.4665   -4.6667
    22         H       -1.0614    0.6275   -5.1945
    23         H        0.8206    0.6805   -3.6635
    24         H        1.5611    1.0023   -1.3501
    25         H        1.0382    2.9238    0.5417
    26         H        1.7642    3.2358    2.8611

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C12 H12 N2  =    26 atoms



      MOLECULAR POINT GROUP   :   C2  


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 35


 -------------------------------------------------------------------------------
PM7 NOOPT  EPS=80.1  CHARGE=0
 201569088026230237782806235680973020929



     NO PARAMETERS MARKED FOR OPTIMIZATION, SO 1SCF WAS USED  
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 16.175M)
                                                       Wed May 10 12:35:58 2017
                                                       No. of days remaining =  45




          FINAL HEAT OF FORMATION =         31.88946 KCAL/MOL =     133.42551 KJ/MOL
          VAN DER WAALS AREA      =        227.07 SQUARE ANGSTROMS


          TOTAL ENERGY            =      -2008.75443 EV
          ELECTRONIC ENERGY       =      -2929.93505 EV  POINT GROUP:     C2  
          CORE-CORE REPULSION     =        921.18062 EV
          DIELECTRIC ENERGY       =         -0.66114 EV
          COSMO AREA              =        227.07 SQUARE ANGSTROMS
          COSMO VOLUME            =        234.69 CUBIC ANGSTROMS

          IONIZATION POTENTIAL    =          8.516218 EV
          HOMO LUMO ENERGIES (EV) =         -8.516 -0.238
          NO. OF FILLED LEVELS    =         35
          MOLECULAR WEIGHT        =        184.2402

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H    22    H    15    10.93170
            H    20    H    26     4.74449
            H    23    H    19     2.36150


          SCF CALCULATIONS        =          2
 
          WALL-CLOCK TIME         =          0.020 SECONDS
          COMPUTATION TIME        =          0.055 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       N          1.23900000      1.62230000      4.91650000   
     2       C          0.91800000      1.38440000      3.57550000   
     3       C          0.11160000      0.30140000      3.22540000   
     4       C         -0.31250000      0.12190000      1.90310000   
     5       C          0.02530000      1.04570000      0.90310000   
     6       C         -0.41890000      0.86120000     -0.48780000   
     7       C         -1.76990000      0.64560000     -0.79740000   
     8       C         -2.18840000      0.46990000     -2.12200000   
     9       C         -1.27640000      0.54630000     -3.17470000   
    10       N         -1.67960000      0.25290000     -4.48200000   
    11       C          0.07660000      0.69900000     -2.87180000   
    12       C          0.49880000      0.87650000     -1.54870000   
    13       C          0.78640000      2.16300000      1.27760000   
    14       C          1.20740000      2.33960000      2.60120000   
    15       H          1.28280000      0.78040000      5.48180000   
    16       H          2.05370000      2.21430000      5.04320000   
    17       H         -0.19390000     -0.42010000      3.97820000   
    18       H         -0.91330000     -0.75200000      1.66010000   
    19       H         -2.51420000      0.61210000     -0.00450000   
    20       H         -3.23900000      0.27650000     -2.32040000   
    21       H         -2.65460000      0.46650000     -4.66670000   
    22       H         -1.06140000      0.62750000     -5.19450000   
    23       H          0.82060000      0.68050000     -3.66350000   
    24       H          1.56110000      1.00230000     -1.35010000   
    25       H          1.03820000      2.92380000      0.54170000   
    26       H          1.76420000      3.23580000      2.86110000   

                             CARTESIAN COORDINATES

   1    N        1.239000000     1.622300000     4.916500000
   2    C        0.918000000     1.384400000     3.575500000
   3    C        0.111600000     0.301400000     3.225400000
   4    C       -0.312500000     0.121900000     1.903100000
   5    C        0.025300000     1.045700000     0.903100000
   6    C       -0.418900000     0.861200000    -0.487800000
   7    C       -1.769900000     0.645600000    -0.797400000
   8    C       -2.188400000     0.469900000    -2.122000000
   9    C       -1.276400000     0.546300000    -3.174700000
  10    N       -1.679600000     0.252900000    -4.482000000
  11    C        0.076600000     0.699000000    -2.871800000
  12    C        0.498800000     0.876500000    -1.548700000
  13    C        0.786400000     2.163000000     1.277600000
  14    C        1.207400000     2.339600000     2.601200000
  15    H        1.282800000     0.780400000     5.481800000
  16    H        2.053700000     2.214300000     5.043200000
  17    H       -0.193900000    -0.420100000     3.978200000
  18    H       -0.913300000    -0.752000000     1.660100000
  19    H       -2.514200000     0.612100000    -0.004500000
  20    H       -3.239000000     0.276500000    -2.320400000
  21    H       -2.654600000     0.466500000    -4.666700000
  22    H       -1.061400000     0.627500000    -5.194500000
  23    H        0.820600000     0.680500000    -3.663500000
  24    H        1.561100000     1.002300000    -1.350100000
  25    H        1.038200000     2.923800000     0.541700000
  26    H        1.764200000     3.235800000     2.861100000


           Empirical Formula: C12 H12 N2  =    26 atoms



      MOLECULAR POINT GROUP   :   C2  


                  EIGENVALUES  
 -36.48702 -36.48523 -30.72980 -30.08464 -26.68825 -26.17471 -25.76164 -24.16777
 -21.95021 -21.46729 -21.10805 -20.04418 -18.27101 -17.42454 -17.40276 -16.95913
 -16.31302 -14.97454 -14.88300 -14.80979 -14.67767 -14.37376 -14.30713 -14.11116
 -13.91632 -13.56514 -12.28020 -12.15858 -12.14318 -11.67799 -11.33486 -10.11358
 -10.08455  -9.18228  -8.51622  -0.23769  -0.06917  -0.03677   0.41016   2.16589
   2.41206   3.08060   3.14785   3.84794   4.05156   4.18791   4.43966   4.64825
   4.67184   4.88863   5.00834   5.02256   5.04199   5.11269   5.62184   5.62965
   5.79994   5.87853   6.00972   6.02174   6.02569   6.11839   6.18490   6.19057
   6.22081   6.24011   6.83454   6.83551


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          N          -0.631004        5.6310     1.59957     4.03143
    2          C           0.206923        3.7931     1.05695     2.73613
    3          C          -0.288445        4.2884     1.07706     3.21139
    4          C          -0.094778        4.0948     1.09095     3.00383
    5          C          -0.088096        4.0881     1.05812     3.02997
    6          C          -0.088082        4.0881     1.05814     3.02994
    7          C          -0.094885        4.0949     1.09097     3.00392
    8          C          -0.288607        4.2886     1.07702     3.21159
    9          C           0.206234        3.7938     1.05700     2.73676
   10          N          -0.630007        5.6300     1.59941     4.03060
   11          C          -0.287570        4.2876     1.07721     3.21036
   12          C          -0.094256        4.0943     1.09125     3.00301
   13          C          -0.094539        4.0945     1.09123     3.00331
   14          C          -0.287613        4.2876     1.07720     3.21041
   15          H           0.303562        0.6964     0.69644
   16          H           0.302537        0.6975     0.69746
   17          H           0.171190        0.8288     0.82881
   18          H           0.164298        0.8357     0.83570
   19          H           0.164325        0.8357     0.83567
   20          H           0.171078        0.8289     0.82892
   21          H           0.303306        0.6967     0.69669
   22          H           0.302396        0.6976     0.69760
   23          H           0.171442        0.8286     0.82856
   24          H           0.164581        0.8354     0.83542
   25          H           0.164537        0.8355     0.83546
   26          H           0.171473        0.8285     0.82853
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.716     0.479    -0.292     0.910
 HYBRID         0.517     0.349    -0.212     0.659
 SUM            1.233     0.828    -0.504     1.569


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  N   1.59957   1.50542   1.35268   1.17333
    2  C   1.05695   0.93321   0.95729   0.84563
    3  C   1.07706   1.08319   1.07550   1.05270
    4  C   1.09095   0.99214   1.03784   0.97386
    5  C   1.05812   1.04672   1.01042   0.97283
    6  C   1.05814   0.98704   1.07304   0.96986
    7  C   1.09097   1.02278   0.95101   1.03012
    8  C   1.07702   1.09371   1.12584   0.99204
    9  C   1.05700   0.97194   0.90012   0.86470
   10  N   1.59941   1.22939   1.66699   1.13421
   11  C   1.07721   1.02883   1.12472   1.05681
   12  C   1.09125   1.07988   0.95102   0.97210
   13  C   1.09123   0.95990   1.02216   1.02126
   14  C   1.07720   1.12432   1.08501   1.00109
   15  H   0.69644
   16  H   0.69746
   17  H   0.82881
   18  H   0.83570
   19  H   0.83567
   20  H   0.82892
   21  H   0.69669
   22  H   0.69760
   23  H   0.82856
   24  H   0.83542
   25  H   0.83546
   26  H   0.82853

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.03 SECONDS

 == MOPAC DONE ==
